zhaquirks/schneiderelectric/__init__.py

```diff
@@ -0,0 +1,135 @@
+"""Quirks implementations for Schneider Electric devices."""
+from typing import Any, Coroutine, Dict, Union
+
+from zigpy import types as t
+from zigpy.quirks import CustomCluster
+from zigpy.zcl import foundation
+from zigpy.zcl.clusters.closures import WindowCovering
+from zigpy.zcl.clusters.general import Basic
+from zigpy.zcl.foundation import ZCLAttributeDef
+
+SE_MANUF_NAME = "Schneider Electric"
+SE_MANUF_ID = 0x4190
```

Comment:
I think this is the wrong manufacturer ID.  It should be `0x105e` which is `4190` in decimal (note the lack of `0x` prefix)

Reply:
Ha, good catch! I’ve corrected that.